course/pages/dagster-essentials/lesson-3/viewing-run-details.md

```diff
@@ -45,23 +43,3 @@ Below the run stats is the Run timeline:
 The timeline visualizes how long each step in the run took to complete and whether it finished successfully or encountered an error. In this example, the `taxi_trips_file` step - the only step in this particular run - is highlighted in green, indicating it finished successfully.
 
 Using the toggles on the left side and above the graph, you can change how this displays depending on your needs.
-
----
-
-## Run logs
```

Comment:
Opting to remove this section since doesn't add to the overall progression and there has been some confusion before.